client/state/data-layer/wpcom/posts/revisions/index.js

```diff
@@ -31,10 +33,15 @@ export function normalizeRevision( revision ) {
 	}
 
 	return {
-		...revision,
+		...omit( revision, [ 'title', 'content', 'excerpt', 'date', 'date_gmt', 'modified', 'modified_gmt' ] ),
 		...flow(
 			pick( [ 'title', 'content', 'excerpt' ] ),
 			mapValues( ( val = {} ) => val.rendered )
+		)( revision ),
+		...flow(
+			pick( [ 'date_gmt', 'modified_gmt' ] ),
+			mapValues( val => `${ val }Z` ),
+			mapKeys( key => key.slice( 0, -'_gmt'.length ) )
```

Comment:
Minor: Readability: I think it's nice that this isn't a magic number, but at the same time the `-` is a bit foreign at first glance. Wondering if either `-1 * '_gmt'.length` or `-4 /* "_gmt" */` would be more readable. What do you think?